src/MerchantCenter/AccountService.php

```diff
@@ -469,6 +473,26 @@ private function maybe_add_merchant_center_url( int $merchant_id ) {
 		}
 	}
 
+	/**
+	 * Get the callback function for linking an Ads account.
+	 *
+	 * @throws Exception When the merchant account hasn't been set yet.
+	 */
+	private function link_ads_account() {
+		if ( ! $this->options->get_merchant_id() ) {
+			throw new Exception( 'A Merchant Center account must be connected' );
+		}
+
+		if ( ! $this->options->get_ads_id() ) {
+			// Return early if the ads account hasn't been created yet.
+			return;
```

Comment:
Same as previous comment, do we need to mark this as STEP_SKIPPED?

Reply:
Responded [here](https://github.com/woocommerce/google-listings-and-ads/pull/2205#discussion_r1500782920)